src/main.ts

```diff
@@ -25,13 +29,21 @@ async function main() {
         let servicePrincipalKey = secrets.getSecret("$.clientSecret", true);
         let tenantId = secrets.getSecret("$.tenantId", false);
         let subscriptionId = secrets.getSecret("$.subscriptionId", false);
+        const enablePSSession = core.getInput('enable-PSSession').toLowerCase() === "true";

```

Comment:
input here should be renamed if that is the case.

Reply:
will rename once its confirmed